package.json

```diff
@@ -1,159 +1,166 @@
-				},
-				"perforce.editOnFileModified": {
-					"type": "boolean",
-					"default": false,
-					"description": "Automatically open a file for edit when Modified"
-				},
-				"perforce.addOnFileCreate": {
-					"type": "boolean",
-					"default": false,
-					"description": "Automatically Add a file to depot when Created"
-				},
-				"perforce.deleteOnFileDelete": {
-					"type": "boolean",
-					"default": false,
-					"description": "Automatically delete a file from depot when deleted"
-				},
-				"perforce.command": {
-					"type": "string",
-					"default": "none",
-					"description": "Configure a path to p4 or an alternate command if needed"
-				}
-			}
-		},
-		"commands": [
-			{
-				"command": "perforce.menuFunctions",
-				"title": "Show Perforce functions",
-				"category": "Perforce"
-			},
-			{
-				"command": "perforce.showOutput",
-				"title": "Show Output",
-				"category": "Perforce"
-			},
-			{
-				"command": "perforce.add",
-				"title": "add - Open a new file to add it to the depot",
-				"category": "Perforce"
-			},
-			{
-				"command": "perforce.edit",
-				"title": "edit - Open an existing file for edit",
-				"category": "Perforce"
-			},
-			{
-				"command": "perforce.revert",
-				"title": "revert - Discard changes from an opened file",
-				"category": "Perforce"
-			},
-			{
-				"command": "perforce.diff",
-				"title": "diff - Display diff of client file with depot file",
-				"category": "Perforce"
-			},
-			{
-				"command": "perforce.diffRevision",
-				"title": "diff revision - Display diff of client file with depot file at a specific revision",
-				"category": "Perforce"
-			},
-			{
-				"command": "perforce.info",
-				"title": "info - Display client/server information",
-				"category": "Perforce"
-			},
-			{
-				"command": "perforce.opened",
-				"title": "opened - Display 'open' files and open one in the editor",
-				"category": "Perforce"
-			}
-		],
-		"keybindings": [
-			{
-				"key": "alt+p space",
-				"command": "perforce.menuFunctions"
-			},
-			{
-				"key": "alt+p l",
-				"command": "perforce.showOutput"
-			},
-			{
-				"key": "alt+p a",
-				"command": "perforce.add",
-				"when": "editorTextFocus"
-			},
-			{
-				"key": "alt+p e",
-				"command": "perforce.edit",
-				"when": "editorTextFocus"
-			},
-			{
-				"key": "alt+p r",
-				"command": "perforce.revert",
-				"when": "editorTextFocus"
-			},
-			{
-				"key": "alt+p d",
-				"command": "perforce.diff",
-				"when": "editorTextFocus"
-			},
-			{
-				"key": "alt+p i",
-				"command": "perforce.info"
-			},
-			{
-				"key": "alt+p o",
-				"command": "perforce.opened"
-			}
-		]
-	},
-	"dependencies": {
-		"tmp": "0.0.28"
-	},
-	"devDependencies": {
-		"vscode": "~0.11.14"
-	}
-}
+{
+  "name": "perforce",
+  "displayName": "Perforce for VS Code",
+  "description": "Perforce basic commands for Visual Studio Code",
+  "version": "0.2.0",
+  "publisher": "slevesque",
+  "license": "LICENSE.md",
+	"categories": [
```

Comment:
I don't mind changing the whole project to use `Spaces: 2` instead of `Tab: 4`, but there is some inconsistency in this file